src/sass/_theme_font_local.sass

```diff
@@ -0,0 +1,6 @@
+$lato-font-path: "~lato-font/fonts"
+$lato-variants: ('normal': 400, 'bold': 700)
```

Comment:
I'm not sure if this here is enough. Someone with more understanding of what fonts are required might need to make an adjustment to this file to bring it into compatibility with previous results.

Reply:
We definitely use the italics one as well. Bolditalic is probably unused.

Reply:
I think we should include it.
Lato is the font for the content. The combination of Sphinx, extensions, and custom CSS could generate a need for a bolditalic fontface. If this is done using a separate `@font-face` rule it will only be downloaded when needed. If it is not needed, it will just be 1 line in CSS, so quite inexpensive.

Reply:
So what would I need to change here, (this is not my forte).